cdap-client-tests/src/test/java/io/cdap/cdap/client/MetadataHttpHandlerTestRun.java

```diff
@@ -1259,7 +1259,7 @@ private void assertDataEntitySearch() throws Exception {
     DatasetId dsWithSchema = NamespaceId.DEFAULT.dataset(AllProgramsApp.DS_WITH_SCHEMA_NAME);
 
     // schema search for a field with the given fieldname:fieldtype
-    assertSearch(searchMetadata(NamespaceId.DEFAULT, "body:STRING+field1:STRING"), dsWithSchema);
+    assertSearch(searchMetadata(NamespaceId.DEFAULT, "body:STRING field1:STRING"), dsWithSchema);
```

Comment:
can you also add a test case where the query contains a plus sign? Because that is now a supported syntax.

Reply:
Added in the latest commit.